### DISCUSSION TITLE
Problem with Test(Lesson 7)

### DISCUSSION THREAD

**Discussion by arbdev95** [Cyfrin/foundry-full-course-cu#1115] [Q&A] [answered]
Hello guys,

hope you're well.

So, following Patrick's course, i stuck when i go to test the "priceFeedVersionIsAccurate"; foundry it says me:

''
forge test -m testPriceFeedVersionIsAccurate
error: unexpected argument '-m' found

Usage: forge test [OPTIONS]

For more information, try '--help'.

And i'm following what Patrick is saying;so what is the problem?

best regards

**Comment by jivan1** [comment, marked as answer]
-m is depreciated, so use --mt instead.
